config/jobs/kubernetes-csi/gen-jobs.sh

```diff
@@ -329,8 +329,8 @@ snapshotter_version() {
         #
         # Additional jobs could be created to cover version
         # skew, if desired.
-        if [ "$kubernetes" == "latest" ] || [ "$kubernetes" == "master" ] || version_gt "$kubernetes" 1.19; then
-            echo '"v4.0.0"'
+        if [ "$kubernetes" == "latest" ] || [ "$kubernetes" == "master" ] || version_gt "$kubernetes" 1.20; then
+            echo '"v6.1.0"'
         else
             echo '"v3.0.3"'
```

Comment:
I don't think we need snapshot v3.0.3 any more.  We can just do "echo v6.1.0" in the else block.

Reply:
done